app/lib/ai/models/vertex-ai/index.js

```
@@ -0,0 +1,89 @@
+        const { keys, scopes } = this.config;
+        const url = `https://dns.googleapis.com/dns/v1/projects/${keys?.project_id}`;
+
+        const client = new JWT({
+            email: keys?.client_email,
+            key: keys?.private_key,
+            scopes,
+        });
+
+        try {
+            const tokens = await client.request({ url });
+            const { Authorization } = tokens?.config?.headers;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)